scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -311,11 +315,13 @@ class FossId(
                     }
                 }
 
-                checkScan(scanCode)
-                val rawResults = getRawResults(scanCode)
-                val resultsSummary = createResultSummary(startTime, rawResults)
+                if (!noWait) {
```

Comment:
I stumbled over this double negation. Do you think it makes sense to use positive logic and rename `noWait` to something like `waitForResult`? Or is `noWait` a term coined by FOSSID itself?

Reply:
No `noWait` comes from me :)
But you are right I will address it.